apps/src/lib/levelbuilder/unit-editor/UnitEditor.jsx

```diff
@@ -286,6 +288,21 @@ class UnitEditor extends React.Component {
       }
     }
 
+    if (
+      this.state.unitPublishedState !== this.props.initialUnitPublishedState
+    ) {
+      const msg =
+        'It looks like you are updating the published state of this unit. ' +
+        'Are you sure you want to update the published state? ';
```

Comment:
this doesn't seem right to me -- I get this error after I check "Hide this unit within this course", so I would expect to see a prompt which uses the same language.

Reply:
I think this is opening a prompt window to check that you want to change the published state of the unit when you are saving. Is that not what you are seeing?

Reply:
Ah I see what you are saying. I will update the message